runner/common/logprob.go

```diff
@@ -7,6 +7,19 @@ import (
 	"github.com/ollama/ollama/llm"
 )
 
+func stringToByteInts(s string) []int {
```

Comment:
I _think_ you might be able to use `[]byte(s)` instead of this function. However when returning it in the API you'd need to make sure it converts to the integer values instead of a byte slice – I think you could create at type with a custom marshal function instead

Reply:
Sorry yes forgot to mention. Did consider this but []byte is b64 in go which is not the desired format